src/tensor_ops/min_to/min_to.cu

```diff
@@ -10,15 +10,38 @@ __device__ __forceinline__ float atomicMinf(float * addr, float value) {
     }
 }
 
+// atomicMax is not implemented for double,
+// solution copied https://stackoverflow.com/questions/17399119/how-do-i-use-atomicmax-on-floating-point-values-in-cuda
+__device__ __forceinline__ double atomicMinf(double * addr, double value) {
+    if (signbit(value)) {
+        return __uint_as_double(atomicMax((unsigned int *)addr, __double_as_uint(value)));
+    } else {
+        return __int_as_double(atomicMin((int *)addr, __double_as_int(value)));
+    }
+}
```

Comment:
This & the max version currently don't work because these intrinsics don't work.

There are the __double2hiint and __double2loint, but then the comparison operations wouldn't work

Reply:
Why not convert to `long long int` instead using [__double_as_longlong](https://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH__INTRINSIC__CAST.html#group__CUDA__MATH__INTRINSIC__CAST_1geb72123ab9c35d88842baf34140a8ca6) and [__longlong_as_double](https://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH__INTRINSIC__CAST.html#group__CUDA__MATH__INTRINSIC__CAST_1g354a51915a0c6465ac5e5c844724d987)?

Reply:
Oh! I totally missed those somehow lmao. Great find!!